web/packages/teleterm/src/mainProcess/types.ts

```diff
@@ -276,3 +276,5 @@ export enum MainProcessIpc {
 export enum WindowsManagerIpc {
   SignalUserInterfaceReadiness = 'windows-manager-signal-user-interface-readiness',
 }
+
+export const GRACEFUL_KILL_MESSAGE = 'GRACEFUL_KILL';
```

Comment:
How about `TERMINATE_MESSAGE` instead? SIGKILL just ends a process with no hesitation, the graceful version of it has a name and it's SIGTERM. ;)

Also, please add a JSDoc explaining what the message signals.